Snakes.py

```diff
@@ -8,15 +8,16 @@
 gameDisplay = pygame.display.set_mode((800,600))
 
 pygame.display.set_caption('SNAKES')
-img=pygame.image.load('Assets\Images\head.png')
```

Comment:
Why are you modifying this line. I don't think this will work

Reply:
It is difficult for me to set up paths every time. So I keep the images in the folder containing the py file.

Reply:
Sir, It at least works on my machine.

Reply:
> Sir, It at least works on my machine.

This simply means that the issue is in your end. You must be doing something wrong.